File: 01.py
class Solution(object):
    def twoSum(self, nums, target):
        z = []
        for j,i in enumerate(nums):
            v = j
            for k in nums[j+1:len(nums)]:
                v+=1
                if int(i)+int(k) == target:
                    return[j,v]

File: javascript/01.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
  for (let j = 0; j < nums.length; j++) {
    for (let k = j + 1; k < nums.length; k++) {
      if (parseInt(nums[j]) + parseInt(nums[k]) == target) {
        return [j, k];
      }
    }
  }
  return;
};

console.log(twoSum([2, 7, 11, 15], 9));
